### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by daisyyjelagat**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by daisyyjelagat — COMMENTED** [review]
Creating a pull request can seem intimidating.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (DaisyKirui) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment (DaisyKirui) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment (DaisyKirui) [inline index.html:50]: <h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
```
Comment (daisyyjelagat) [inline index.html]:
```suggestion
GAME OVER ! TRY AGAIN!
```

**Pull request merged**